supervisely/cli/project/project_download.py

```diff
@@ -3,13 +3,23 @@
 from rich.console import Console
 from tqdm import tqdm
 import supervisely as sly
+from dotenv import load_dotenv
+import os
 
 
 def download_run(id: int, dest_dir: str) -> bool:
-
-    api = sly.Api.from_env()
     console = Console()
 
+    load_dotenv(os.path.expanduser("~/supervisely.env"))
+    try:
+        api = sly.Api.from_env()
+    except ValueError as e:
```

Comment:
How to handle ValueError?

Reply:
def __handle_creds_error_to_console(api_func):
      try:
         api = api_func()
     except Exception as e:
         console = Console()
         if "SERVER_ADDRESS env variable is undefined" in repr(e):
              console.print(SERVER_ADDRESS env variable is undefined. Add it to your "~/supervisely.env" file or to environment variables. Learn more here: https://developer.supervise.ly/getting-started/basics-of-authentication. )
          elif "API_TOKEN env variable is undefined":
           console.print( .... 
          else: 
              console.print(repr(e))
              exit(1)
      return api

api = _handle_error_to_console(sly.Api.from_env)